src/assignment/assignment.helper.tsx

```diff
@@ -164,11 +164,11 @@ export class AssignmentHelper {
 							label={i18n.t('assignment/views/assignment-edit___klas-of-groep')}
 							required
 						>
-							<TextInput
-								id="class_room"
-								value={/*assignment.class_room || */ 'TODO CLASSroom was changes'}
-								onChange={(classRoom) => setAssignmentProp('class_room', classRoom)}
-							/>
+							{/*<TextInput*/}
+							{/*	id="class_room"*/}
+							{/*	value=/!*assignment.class_room || 'TODO CLASSroom was changes'}*/}
+							{/*	onChange={(classRoom) => setAssignmentProp('class_room', classRoom)}*/}
+							{/*/>*/}
```

Comment:
For future reference; deze helper zal op termijn verdwijnen want dit hele formulier is niet meer relevant. Aarzel dus niet om gewoon de code weg te gooien als er typings klagen in de toekomst. ✌️